### DISCUSSION TITLE
Possible Error With Character Color

### DISCUSSION THREAD

**Discussion by Josh-Jo1** [starship/starship#5369] [Q&A]
Hello all! I have a question that may or may not be a bug. The following is from Powershell 7, using starship v1.16.0:
![image](https://github.com/starship/starship/assets/55329168/d1a694cc-128a-4679-9cc2-5aa72d59772e)

In this case, shouldn't the 3rd `$character` (i.e. '>') be red, since the previous command failed? Notice that it did work for the 5th one though.

I have not set anything for `$character` in my `starship.toml`, so it uses the default values.

**Comment by chipbuster** [comment]
This does indeed look like a bug. I'm going to go ahead and close this discussion so that we can keep discussion on the primary issue at #5376
